include/service/entities/site_query_service.js

```diff
@@ -260,13 +260,20 @@ module.exports = function SiteQueryServiceModule(pb) {
    * @param {Function} cb
    */
   SiteQueryService.prototype.getCollections = function (cb) {
-    this.listCollections({ name: { $nin: ["system.indexes", "system.namespaces"] }}, function(err, items) {
-      if(pb.util.isError(err)) {
-        pb.log.error(err);
-      }
-      cb(err, items);
-    });
-  };
+      this.listCollections({}, function(err, items) {
```

Comment:
The issue we ran into with handing the filtering off to the db was that it was dependent on several things, from your version of node to mongod npm package you had installed. taking a slightly less efficient route that guaranteed the proper results seemed like the best way to solve the problem, and it did so we figured we would push it back to you so others dont run into the same issue.